stratum/hal/bin/barefoot/docker/build-stratum-bf-container.sh

```diff
@@ -61,9 +61,12 @@ if [ -n "$1" ]; then
       ((i+=1))
   done
   # Mount BSP folder and pass it to the build script, if requested.
```

Comment:
If checks looks good now, can you update the comment to mention the new way?
Something like 
```
# Mount BSP folder and pass it to the build script, if requested, for SDE 9.6.0 and before.
# Pass in the BSP tarball directly, starting with SDE 9.7.0
```